### PR TITLE
Added shanecsj.adoc, updated AboutUs and UserGuide

### PR DESCRIPTION
**Pull request by shanecsj**
- Added shanecsj.adoc
- Added shanecsj.png
- Updated AboutUs.adoc
- Updated cookbook commands in UserGuide

### CODE REVIEW

**Review by tzihiang — APPROVED** [review]

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -129,9 +129,25 @@ This command allows you to view a recipe, the ingredients required, and the step
 
 - Format: `cookbook view recipe INDEX`
 - Example:
-* `cookbook view recipe 1` +
+* `cookbook view 1` +
     Views the ingredients and steps of recipe 1.
 
+==== Add a new recipe to the cookbook
```
Comment (gohkh) [inline docs/UserGuide.adoc:135]: Maybe can add an optional description argument?

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -129,9 +129,25 @@ This command allows you to view a recipe, the ingredients required, and the step
...
+
+- Format: `cookbook add n/NAME`
+- Example:
+* `cookbook add n/Bacon Carbonara` +
+    Adds a new recipe with the name 'Bacon Carbonara'.
+
+==== Delete a recipe from the cookbook
+This command allows you to delete a recipe of the specified index from the cookbook.
+
+- Format: `cookbook delete INDEX`
```
Comment (gohkh) [inline docs/UserGuide.adoc]: Are we standardising the command to "remove"?

Comment (shanecsj) [reply docs/UserGuide.adoc]: My bad. Will edit accordingly in the next commit.

File: docs/UserGuide.adoc (hunk 1 above)
Comment (shanecsj) [reply docs/UserGuide.adoc:135]: Will add an optional argument for ingredients, steps and tags.

**Review by hans555 — APPROVED** [review]

**Review by teo-jun-xiong — APPROVED** [review]

**Review by gohkh — APPROVED** [review]

**Pull request merged**